### ISSUE TITLE
Impact-Subpack and Reactor-Masking-Helper (`weights_only=True`) error

### ISSUE THREAD

**Issue by Buumcode** [ltdrdata/ComfyUI-Impact-Subpack#61]
1/ Clean installation of of Reactor --> Masking Helper node works well, without any error.
2/ Added installation of Impact Pack and Impact Subpack --> Mask Helper now not work, return error ("weights_only") --
**WeightsUnpickler error: Unsupported global: GLOBAL ultralytics.nn.tasks.DetectionModel was not an allowed global by default.**
ComfyUI version: 0.3.39
ComfyUI frontend version: 1.21.3
ComfyUI-Impact-Pack (V8.17.1)
ComfyUI-Impact-Subpack (V1.3.3)
[ReActor] - STATUS - Running v0.6.1-b2 in ComfyUI -- freedom version from [Codeberg.org](https://codeberg.org/Gourieff/comfyui-reactor-node)
3/ Adding in model-whitelist.txt all relayted models -- do nothing. Error still persist.
Added to model-whitelist.txt (all exist 'yolo' models and couple others):
skin_yolov8m-seg_400.pt
face_yolov8m-seg_60.pt
yolov5m.pt
yolov5n-face.pth
yolov5l-face.pth
face_yolov8s.pt
person_yolov8s-seg.pt
person_yolov8m-seg.pt
hand_yolov8s.pt
face_yolov8m.pt
sam_hq_vit_l.pth
sam_vit_b_01ec64.pth
sam_vit_l_0b3195.pth
4/ Working solution: you need to modify *\comfyui-impact-subpack\modules\subcore.py file (and one Reactor's file -- nodes.py) to add some code (use Notepad++ compare plug-in to see what changed). Only after that problem with Impact-Subpack will be solved -- [look here](https://codeberg.org/Gourieff/comfyui-reactor-node/issues/4#issuecomment-3038859) or here https://github.com/Gourieff/ComfyUI-ReActor/issues/73#issuecomment-2726569177

**Added code to subcore.py:
+from torch.nn.modules.container import Sequential
+from ultralytics.nn.tasks import DetectionModel
+restricted_getattr, dill._dill._load_type, scalar, dtype, Float64DType, DetectionModel, Sequential]

**Comment by ltdrdata** [comment]
What is your `ultralytics` package version?
For some packages, the import path varies depending on the version, so it needs to be checked for reproducibility.

**Comment by Buumcode** [comment]
>What is your ultralytics package version?

`pip list `

Microsoft Windows [Version 10.0.19045.5965]
Package                        Version
------------------------------ ------------------------
<...>
ultralytics                    8.3.40
ultralytics-thop               2.0.13
<...>

**Comment by ltdrdata** [comment]
There was a recent patch related to this. Try updating Ultralytics to the latest version. (`ultralytics==8.3.162`)

**Comment by Buumcode** [comment]
>Try updating Ultralytics to the latest version. (ultralytics==8.3.162)

OK, may be that will works. My Ultralytics has version, which is got when I update ComfyUI to version 0.3.39 - and run "pip install -r requirements.txt" for this version.

**Comment by Buumcode** [comment]
@ltdrdata 

Ok, it's works. ✌

--- Experiment ---
1. Fresh new install ComfyUI v0.3.43 (with apply requirements.txt) -----> ultralytics 8.3.40
2. Fresh new install Reactor v0.6.1-b2  (with apply requirements.txt) -----> ultralytics 8.3.40 ------> Masking Helper works well
3. Fresh new install Impact Pack v8.18  (with apply requirements.txt)  -----> ultralytics 8.3.40 ------> Masking Helper works well
4. Fresh new install Impack Subpack v1.3.4  (with apply requirements.txt) -----> ultralytics 8.3.40 -----> ERROR!!! (weights_only=true)

```
pip install -U ultralytics
```

Installing collected packages: ultralytics
  Attempting uninstall: ultralytics
    Found existing installation: ultralytics 8.3.40
    Uninstalling ultralytics-8.3.40:
      Successfully uninstalled ultralytics-8.3.40
Successfully installed ultralytics-8.3.162

---------> Masking Helper works well! ✅
